packages/store/addon/-private/system/backburner.js

```diff
@@ -6,6 +6,7 @@ import { registerWaiter } from '@ember/test';
 import { DEBUG } from '@glimmer/env';
 import Ember from 'ember';
 
+// TODO: expose Ember._Backburner as `import { _Backburner } from '@ember/runloop'` in ember-rfc176-data + emberjs/ember.js
```

Comment:
Can you please clarify this a bit, not quite sure what the future work required is

Reply:
Ember doesn't expose the constructor for Backburner in a modules API